cmd/collectors/rest/plugins/volume/volume.go

```diff
@@ -155,6 +160,10 @@ func (v *Volume) updateVolumeLabels(data *matrix.Matrix, volumeMap map[string]vo
 					v.Logger.Error().Err(err).Str("metric", "cloneSplitEstimateMetric").Msg("Unable to set value on metric")
 				}
 			}
+		} else {
+			// Difference between private cli and public api would be node root volumes and temp volumes
```

Comment:
The public API does not include node root and temp volumes, while the private CLI does include them. Harvest will exclude them the same as the public API by not exporting them.